index.html

```diff
@@ -13,43 +13,63 @@
 
     <section class="home-page">
         <header>
-          <nav class="nav">
-            <h1 class="title">My logo</h1>
-            <a class="mob" href="#"><img src="images/Union.png" alt="my logo" /></a> 
-            <ul class="internal-link">
-                <li><a href="#">Portfolio</a></li>
-                <li><a href="#">About</a></li>
-                <li><a href="#"> Contact </a></li>
-             </ul>
-          </nav>
+            <nav class="nav-mob">
+                <a class="title">My logo</a>
+                <a class="icon-menu" href="#"><img src="images/Union.png" alt="my logo" /></a>
+            </nav>
+
+            <ul class="internal-link-mob">
```

Comment:
- [ ] [OPTIONAL] Good job implementing the requested changes, you can give the menu element's some space or remove the margin so they are closer to the edge of the page like in the Figma design

![Screenshot_20220727_210418](https://user-images.githubusercontent.com/25227022/181362618-b49c44c0-ae79-41aa-bf20-4bc4a754af88.png)